aiida/orm/querybuilder.py

```diff
@@ -1766,6 +1766,23 @@ def get_json_compatible_queryhelp(self):
             'offset': self._offset,
         })
 
+    @property
+    def queryhelp(self):
+        """queryhelp dictionary correspondig to QueryBuilder instance.
+
+        The queryhelp can be used to create a copy of the QueryBuilder instance like so::
+
+            qb = QueryBuilder(limit=3).append(StructureData, project='id').order_by({StructureData:'id'})
+            qb2=QueryBuilder(**qb.queryhelp)
+
+        :return: a queryhelp dictionary
+        """
+        return self.get_json_compatible_queryhelp()
+
+    def __deepcopy__(self, memodict={}):
+        qb = type(self)(**self.queryhelp)
```

Comment:
no need for intermediate variable assignment

Reply:
I had it in here as a bit of a help that what is being returned here is actually a querybuilder.
debatable though whether this is helpful - I'll get rid of it